src/vcpkg/commands.upgrade.cpp

```diff
@@ -54,8 +56,16 @@ namespace vcpkg::Commands::Upgrade
 
         const ParsedArguments options = args.parse_arguments(COMMAND_STRUCTURE);
 
```

Comment:
Also, make sure that you check for both `OPTION_KEEP_GOING` and `OPTION_NO_KEEP_GOING` and error if both are passed.